### DISCUSSION TITLE
Problems with the "Upload proof of your academic status" page

### DISCUSSION THREAD

**Discussion by elonWrong** [community/community#113240] [GitHub Education] [answered]
### Hi everyone,

Hey,
So rn i'm encountering a problem with the "Upload proof of your academic status" page. Where every time I try to submit and process a new document, it directs me back to the same page. 
I was rejected once for not completing 2FA, though that has been corrected.
There's no indication that my other attempts were rejected. They just... wouldn't go thru

**Comment by evawolf96** [comment]
I'm having the same problem.

**Reply by winstontsui** [reply]
Refresh the browser or try a smaller image to upload, worked for me

**Reply by btl-monk** [reply]
Upload still not working?

**Comment by pbarni** [comment]
had the same problem, check if your picture is under 1mb

**Comment by SebastienXia** [comment, marked as answer]
The upload option doesn't work, you need to proof your academic status with your camera phone. It's what I do for this problem.

**Reply by elonWrong** [reply]
Although janky with me having to take a pic with my front camera, 
this worked!

**Reply by SebastienXia** [reply]
Nice 👏

**Reply by ItIzYe** [reply]
So my problem is that I have a student card where my name is on the front and the rest on the back of the card so i guess this option won't work for me

**Reply by SebastienXia** [reply]
@ItIzYe The school ID card is one of the ways to prove that you are a student, there are others, such as:
- A 2023-2024 schooling certificate

**Reply by elonWrong** [reply]
@ItIzYe hey do you have an acceptance letter or sth? at least I used that for mine and it worked

**Reply by ItIzYe** [reply]
@elonWrong We usually only get acceptance letters when we join a school so the last on I have is nearly 8years old. But I can ask our secretary if there's a way to get handed a acceptance letter for this year..

**Reply by elonWrong** [reply]
does it say anywhere that you'll be graduating on a future date? for example 2025/2026

**Reply by sysoutAmanX** [reply]
What is the solution to this problem? I have been trying since yesterday! The 'take a picture' option only works with the front camera and cannot capture my card numbers clearly due to bad quality.
![Screenshot 2024-03-30 001521](https://github.com/community/community/assets/126079650/fb7166d2-b029-4fae-8c45-1eca17209659)

**Reply by sysoutAmanX** [reply]
@SebastienXia, could you please help me with this? I would greatly appreciate your assistance.

**Reply by elonWrong** [reply]
@sysoutAmanX What i end up doing is, really using my front camera for my proof. Give it enough light and the picture quality will be acceptable.

**Reply by sysoutAmanX** [reply]
@elonWrong thank you, but it did not work too..

**Reply by elonWrong** [reply]
@sysoutAmanX wait so what did it say? even tho I thought my picture was bad, it still worked

**Reply by sysoutAmanX** [reply]
@elonWrong an error occurred, and they did not accept the picture. 
![WhatsApp Image 2024-04-13 at 19 35 10_04ecb479](https://github.com/community/community/assets/126079650/45d0038a-9bbe-4238-b93c-f5d6386e99e0)

**Reply by elonWrong** [reply]
@sysoutAmanX did u set up 2FA n add your billing information in your name?

**Reply by sysoutAmanX** [reply]
@elonWrong Finally! I am glad that they have fixed it, thanks for the help. 
![image](https://github.com/community/community/assets/126079650/3022f01c-3c28-46c6-b4ee-87c94fc96454)

**Reply by elonWrong** [reply]
@sysoutAmanX nice glad to hear! make sure you do everything correctly on your attempt!

**Reply by MillerAnthony-FS** [reply]
> @elonWrong Finally! I am glad that they have fixed it, thanks for the help. ![image](https://private-user-images.githubusercontent.com/126079650/324282911-3022f01c-3c28-46c6-b4ee-87c94fc96454.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA5Nzc1NTcsIm5iZiI6MTcyMDk3NzI1NywicGF0aCI6Ii8xMjYwNzk2NTAvMzI0MjgyOTExLTMwMjJmMDFjLTNjMjgtNDZjNi1iNGVlLTg3Yzk0ZmM5NjQ1NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxNFQxNzE0MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MWMzNjM0OTQwNGE4NWJhNWZjMGNhNmEyNDMzMzRmZDQ0ZjJlZjMxYjkwZGViNWUxY2ZhZjIwNzMxYzZlZjVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-BlHx04aRKklbqiRFERHrnfDgDbVVrOXWi62H6TDTMo)

How are you seeing the upload option?
I've tried my iPhone, iPad and Windows Laptop. None give that option. 
Help?

**Reply by SamTseng** [reply]
> > @elonWrong Finally! I am glad that they have fixed it, thanks for the help. ![image](https://private-user-images.githubusercontent.com/126079650/324282911-3022f01c-3c28-46c6-b4ee-87c94fc96454.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA5Nzc1NTcsIm5iZiI6MTcyMDk3NzI1NywicGF0aCI6Ii8xMjYwNzk2NTAvMzI0MjgyOTExLTMwMjJmMDFjLTNjMjgtNDZjNi1iNGVlLTg3Yzk0ZmM5NjQ1NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxNFQxNzE0MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MWMzNjM0OTQwNGE4NWJhNWZjMGNhNmEyNDMzMzRmZDQ0ZjJlZjMxYjkwZGViNWUxY2ZhZjIwNzMxYzZlZjVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-BlHx04aRKklbqiRFERHrnfDgDbVVrOXWi62H6TDTMo)
> 
> How are you seeing the upload option? I've tried my iPhone, iPad and Windows Laptop. None give that option. Help?

Me too. I did not see any other options, either using Chrome or Safari on MacBook. I only have one way, that is: "take a picture". And with this way, I failed to upload my photo using my WebCam for each time I tried.

**Reply by zachwolek** [reply]
Did any of y'all find a way to upload it without physically printing and taking a picture of it? 

It just feels like such an antiquated method when I literally have the .pdf to drag and drop

**Reply by savio-mb** [reply]
Any update for a person who have the doc in .pdf? The camera of my phone and webcam is poor.

**Reply by alecchalm** [reply]
Agreed, this is such an awful system for proving academic status... or anything at all. GitHub of all the places, I thought would be able to offer a functioning page.

**Comment by hossanf** [comment]
I still don't see the upload option.. has anyone else had any trouble getting re-verified?

**Reply by ehEheH3heHeh3he** [reply]
I also have this problem
<img width="400" alt="Screenshot 2567-11-30 at 15 11 59" src="https://github.com/user-attachments/assets/89762c79-2589-45dd-9093-482dbb39e1fb">

**Comment by shtsus** [comment]
Okulumun adı açık öğretim fakültesi yani adında bile yazıyor  ama fotoğraf çekerek kanıtlayamıyorum
